src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -45,7 +46,7 @@ function BaseOnboardingAccounting({shouldUseNativeStyles, route}: BaseOnboarding
     const {onboardingIsMediumOrLargerScreenWidth, isSmallScreenWidth, shouldUseNarrowLayout} = useResponsiveLayout();
     const [onboardingValues] = useOnyx(ONYXKEYS.NVP_ONBOARDING);
     const [onboardingPurposeSelected] = useOnyx(ONYXKEYS.ONBOARDING_PURPOSE_SELECTED);
-    const [onboardingPolicyID] = useOnyx(ONYXKEYS.ONBOARDING_POLICY_ID);
+    const [onboardingPolicyID, onboardingPolicyIDResults] = useOnyx(ONYXKEYS.ONBOARDING_POLICY_ID);
```

Comment:
NAB: `onboardingPolicyID` is singular, so the name can be `onboardingPolicyIDResult` here.

Reply:
Done